clients/cloud/rust/src/producer.rs

```diff
@@ -0,0 +1,44 @@
+extern crate futures;
+extern crate rdkafka;
+
+use futures::future::Future;
+use rdkafka::error::KafkaError;
+use rdkafka::message::OwnedMessage;
+use rdkafka::producer::{FutureProducer, FutureRecord};
+use std::boxed::Box;
+
+mod utils;
+
+fn log_produce_result(result: Result<(i32, i64), (KafkaError, OwnedMessage)>) -> Result<(), ()> {
+    result
+        .and_then(|(p, o)| {
+            println!("produced to partition {} with offset {}", p, o);
+            Ok(())
+        })
+        .map_err(|(err, _)| eprintln!("kafka error: {}", err))
+}
+
+fn main() -> Result<(), Box<std::error::Error>> {
+    let (topic, config) = utils::get_config()?;
+    let producer: FutureProducer = config.create()?;
+
+    let messages = (0..9)
+        .map(|msg| {
+            producer.send(
```

Comment:
For consistency with the other client examples, can this print that it's _about to send_?  Specifically, this is what the other clients print:

```bash
Producing record: alice 	 {"count": 0}
...
```